runbotics/runbotics-orchestrator-ui/src/src-app/hooks/useProcessVariables.types.ts

```diff
@@ -0,0 +1,14 @@
+export interface ActionVariableObject {
+    name?: string;
+    value?: string;
+}
```

Comment:
all fields are optional?

Reply:
yes, because returned value has switched name and value values as advised by Franek, to be consistent with the global variables and attended variables

![image](https://user-images.githubusercontent.com/61866178/224939964-bf10058a-0c7d-43e8-9ec0-310b814deba9.png)

Reply:
changed both to be mandatory as if there is an object it always includes keys name and value (can be an empty string)